### ISSUE TITLE
v8 iOS & OS X have no glyphs

### ISSUE THREAD

**Issue by incanus** [mapbox/mapbox-gl-native#2164]
Seeing this as of 92dd26c8be78b30429e4d38bad3b67b32046ed06. 

<img width="1136" alt="screen shot 2015-08-24 at 12 04 29 pm" src="https://cloud.githubusercontent.com/assets/17722/9449582/4ec53fc8-4a58-11e5-938d-40c95b8b03ec.png">
![ios simulator screen shot aug 24 2015 12 04 18 pm](https://cloud.githubusercontent.com/assets/17722/9449583/4ee5d1ac-4a58-11e5-85d6-a4bf9d2b8477.png)

@jfirebaugh

**Comment by friedbunny** [comment]
```objc
2015-08-24 12:13:29.776 Mapbox GL[23540:5101727] [ERROR] {Map}[Style]: Failed to load [mapbox://fonts/mapbox/DIN530003135040656713406ffc                   Pro                   Regular 3Arial 6197065035697329971nicode                   MS                   Regular/0-255.pbf]: unsupported URL
2015-08-24 12:13:29.777 Mapbox GL[23540:5101727] [ERROR] {Map}[Style]: Failed to load [mapbox://fonts/mapbox/DIN530003135040656713406ffc                   Pro                   Medium 3Arial 6197065035697329971nicode                   MS                   Regular/0-255.pbf]: unsupported URL
2015-08-24 12:13:29.778 Mapbox GL[23540:5101727] [ERROR] {Map}[Style]: Failed to load [mapbox://fonts/mapbox/DIN530003135040656713406ffc                   Pro                   Bold 3Arial 6197065035697329971nicode                   MS                   Bold/0-255.pbf]: unsupported URL
```

**Comment by jfirebaugh** [comment]
Possibly related to https://github.com/mapbox/mapbox-gl-native/pull/2141 (cc @lucaswoj).

**Comment by friedbunny** [comment]
Fixed in 648b5879599f6587eb7df36b4e951ec761c6437a.
